### DISCUSSION TITLE
Future To-Do's and possible feature requests

### DISCUSSION THREAD

**Discussion by Szybet** [Quill-OS/quill#38] [Ideas]
- [ ] [Turning off the interface using ioctl ](https://github.com/koreader/koreader/blob/615072c7771ca801bbc1d70154956fac1d1b4b61/platform/kobo/disable-wifi.sh#L62-L66)on the nia and other devices: in toggle.sh and ipd ( propably not needed? )
- [ ] faster database generating
- [-] Easter eggs, replacing icons in specified dates
- [ ] wifi connect dialog: increase height because of landscape
- [ ] Library - proper support for long names
- [ ] IPD: implement appsList update mechanism

**Comment by chloe-everhart** [comment]
Could it have phone calls via signal, if one had a bluetooth conference speaker and wifi?

**Reply by Szybet** [reply]
signal apps are written in typescript, so a browser, so a no no for devices below 1 Gb

**Reply by chloe-everhart** [reply]
due to sight challenges, and known extra challenges opening plastic, i'm concerned i may not succeed at opening the case without breaking it..  i'd like to ask about buying an older, less expensive model with inkbox installed, and if it's got lights, the warmest?  add a store to the website?

**Reply by Szybet** [reply]
we don't sell inkbox os devices
